src/Apps/Sell/Components/SubmissionHeader.tsx

```diff
@@ -199,3 +210,89 @@ export const SubmissionHeader: React.FC = () => {
     </Sticky>
   )
 }
+
+const HeaderArtsyLogo: React.FC<{ withExitConfirmation }> = ({
+  withExitConfirmation,
+}) => {
+  const { router } = useRouter()
+  const { isLoggedIn } = useSystemContext()
+
+  const [showExitConfirmationModal, setShowExitConfirmationModal] = useState(
+    false
+  )
+
+  const handleClick = () => {
+    withExitConfirmation
+      ? setShowExitConfirmationModal(true)
+      : router.push("/sell")
+  }
+
+  // When logged in, it's possible to easily exit the flow by clicking on "Save & Exit"
+  // and the Artsy logo should not be clickable.
+  if (isLoggedIn) {
+    return (
+      <Media greaterThan="xs">
+        <Box data-testid="artsy-logo">
+          <ArtsyLogoIcon display="block" />
+        </Box>
+      </Media>
+    )
+  }
+
+  return (
+    <>
+      <Clickable display="block" onClick={handleClick}>
+        <Box data-testid="artsy-logo">
+          <Media greaterThanOrEqual="sm">
+            <ArtsyLogoIcon display="block" />
+          </Media>
+          <Media lessThan="sm">
+            <ArtsyMarkIcon display="block" />
+          </Media>
+        </Box>
+      </Clickable>
+
+      {showExitConfirmationModal && (
+        <ExitConfirmationModal
+          onClose={() => setShowExitConfirmationModal(false)}
+          onLeave={() => router.push("/sell")}
+        />
+      )}
+    </>
+  )
+}
+
+interface ExitConfirmationModalProps {
+  onClose: () => void
+  onLeave: () => void
+}
+
+export const ExitConfirmationModal: React.FC<ExitConfirmationModalProps> = ({
```

Comment:
We already have an `ConfirmationModalBack` component that has the same styling and copies. It is used in MyCollection Artwork Form.
What do you think about reusing it for this flow? It might require some adjustments

Reply:
That's a very good idea, thanks! The copy is almost the same as well.

Reply:
I've updated the code to use the already existing component from My Collection 👍